azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/Constants.java

```diff
@@ -8,6 +8,11 @@
 
 public class Constants {
     public static final String APP_SETTING_WEBSITE_RUN_FROM_PACKAGE = "WEBSITE_RUN_FROM_PACKAGE";
+    public static final String APP_SETTING_MACHINEKEY_DECRYPTION_KEY = "MACHINEKEY_DecryptionKey";
+    public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE = "WEBSITES_ENABLE_APP_SERVICE_STORAGE";
+    public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE_VALUE = "false";
+    public static final String APP_SETTING_FUNCTION_APP_EDIT_MODE = "FUNCTION_APP_EDIT_MODE";
+    public static final String APP_SETTING_FUNCTION_APP_EDIT_MODE_VALUE = "readOnly";
```

Comment:
APP_SETTING_FUNCTION_APP_EDIT_MODE_READONLY

Reply:
Will fix the naming in later pr.